config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/ingredients", to: "ingredients#index"
+
+  get "/recipes/:id", to: "recipes#show"
+
+  get "/recipes/:id/new", to: "recipe_ingredients#new"
+  post "/recipes/:id", to: "recipe_ingredients#create"
 end
```

Comment:
unRESTful routes. Should be:

`get "/recipes/new", to: "recipes#new` (Again this route is unnecessary for this IC)
`post "/recipes/:id/ingredients", to: "recipe_ingredients#create`